config/logstash/logstash.conf

```diff
@@ -0,0 +1,51 @@
+input {
+  jdbc {
+    jdbc_driver_library => "/usr/share/logstash/postgresql-42.6.0.jar"
+    jdbc_driver_class => "org.postgresql.Driver"
+    jdbc_connection_string => "jdbc:postgresql://db/postgres"
+    jdbc_user => "postgres"
+    jdbc_password => "mysecretpassword"
+    schedule => "* * * * *"
+
+    tracking_column => "id"
+    use_column_value => true
+    statement => "SELECT audit.change.id, audit.change.table_id, audit.change.primary_key, audit.change.foreign_key, audit.change.action, CAST(audit.change.fields AS text) as fields, audit.change.modified_dts, CAST(row_to_json(user_account.*) as text) as modified_by, CONCAT(audit.change.table_id::text, '_', audit.change.primary_key::text) AS guid FROM audit.change LEFT JOIN user_account ON audit.change.modified_by = user_account.id WHERE audit.change.id > :sql_last_value"

```

Comment:
Can you add a comment explaining this query? Things to call out:

- How it determines _what_ to ingest (`sql_last_value`)
- How it pulls user info (use of `user_account` table)